freqtrade/analyze.py

```diff
@@ -172,33 +172,44 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool, sell
         if the threshold is reached and updates the trade record.
         :return: True if trade should be sold, False otherwise
         """
+        current_profit = trade.calc_profit_percent(rate)
+        experimental = self.config.get('experimental', {})
```

Comment:
Just a really minor thing, but I think we can move this below the `stop_loss_reached` condition, to improve the performance for backtesting a little bit

Reply:
good catch